src/main/scala/org/constellation/ConstellationNode.scala

```diff
@@ -203,7 +208,8 @@ case class NodeConfig(
   cliConfig: CliConfig = CliConfig(),
   processingConfig: ProcessingConfig = ProcessingConfig(),
   dataPollingManagerOn: Boolean = false,
-  allocAccountBalances: Seq[AccountBalance] = Seq.empty
+  allocAccountBalances: Seq[AccountBalance] = Seq.empty,
+  storeAllData: Boolean = false
```

Comment:
Could you re-use `constellation.snapshot-size-disk-limit` and set it to 0 (meaning that it's unlimited) general less config properties is better especially in the case that setting 1 property overrides another.